sos/plugins/openstack_instack.py

```diff
@@ -28,6 +30,16 @@ def setup(self):
         self.add_copy_spec("/home/stack/instackenv.json")
         self.add_copy_spec("/home/stack/undercloud.conf")
 
+        # collect undercloud-debug: undercloud-debug.log will be created so we
+        # need to generate it in a tmp.dir, collect it now and then cleanup it
+        tmpdir = gettempdir()
+        self.get_cmd_output_now("su -s /bin/sh -c 'source ~/stackrc && " +
+                                "/usr/libexec/openstack-tripleo/" +
+                                "undercloud-debug' stack",
+                                timeout=600, runat=tmpdir,
+                                suggest_filename="undercloud-debug.log")
+        remove("%s/undercloud-debug.log" % tmpdir)
```

Comment:
I am not sure if we want to put the nova/ironic/neutron/heat requests in the generinc sosreport modules as we usually do not collect this for generic overclouds. This will get a lot of data we usually do not review (at least right now) .

I have created an openstack-instack version based of the undercloud-debu script at:
https://github.com/stuggi/sos/commit/9bedacbb4ceaaa5b749310e928a6c6955ae79b0c

Should I create a new pull request or do we want to handle this here?